examples/configs/recipes/llm/grpo-llama3.2-1b-instruct-1n8g-megatron_generation.yaml

```diff
@@ -0,0 +1,36 @@
+defaults: ../../grpo_math_1B.yaml
```

Comment:
@shanmugamr1992 do you have convergence plots for this recipe? Can you attach those to this PR description?

Reply:
Nope. I can run it and attach it.